### PR TITLE
Feature/INTG-506 Create a change log file for the EspressoPOS project

### PR DESCRIPTION
**Pull request by jwsandeman**
INTG-506 Create a change log file for the EspressoPOS project

### CODE REVIEW

**Comment by rossi-mx51** [comment]
@jwsandeman  looks like you branched off from master to create this branch.  That's why you see all the previous commits here. 
image


Let me know if you don't know how to fix it, I'll give you a hand.

**Review by rossi-mx51 — DISMISSED** [review]
Approved from dev perspective. Need  Product to review as well

File: CHANGELOG.MD [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+### Fixed
+
+- [INTG-504](https://mx51.atlassian.net/browse/INTG-504) Security and stabilty enhancements
+
+## [1.2.4] - 2023-5-10
+
+### Changed
+
+- [INTG-491](https://mx51.atlassian.net/browse/INTG-491) Bump SPI version to 2.9.7
```
Comment (mjancev) [inline CHANGELOG.MD:40]: @jwsandeman where is the bump to use SPI 2.9.8 in the log INTG-512? cc: @rossi-mx51

Comment (mjancev) [reply CHANGELOG.MD:40]: I also can't see the ticket to hide P@T INTG-490

Comment (mjancev) [reply CHANGELOG.MD:40]: or INTG-503 for the override update

**Review by mjancev — CHANGES_REQUESTED** [review]

File: CHANGELOG.MD (hunk 1 above)
Comment (jwsandeman) [reply CHANGELOG.MD:40]: No worries ill have a look. I was just going off the release commits which aren't always comprehensive. I'll check the last couple of sprints' worth of tickets as well.

**Review by mjancev — APPROVED** [review]
LGTM

**Review by rossi-mx51 — APPROVED** [review]

**Pull request merged**